ja.po: app_crypt.xml

amotoki opened the pull request.

amotoki committed:
ja.po: app_crypt.xml

takekawa commented on doc/src/docbkx/openstack-ops/locale/chap/app_crypt.xml.ja.po:
```diff
@@ -0,0 +1,992 @@
+msgstr ""
+"私は、新しい OpenStack クラウドのセットアップをする為、カナダのブリティッシュ"
+"コロンビア州ケロウナの現地にいた。デプロイ作業は完全に自動化されていた。"
+"Cobbler は物理マシン上の OS をデプロイし、それを起動し、その後は Puppet が引"
```
Cobbler は物理マシン上の OS をデプロイし
は
Cobbler で物理マシン上の OS をデプロイし
ではないでしょうか?
参考：Cobbler http://www.cobblerd.org/

irixjp replied:
Cobbler は物理マシンにOS をデプロイし、

って感じ？

takekawa commented on doc/src/docbkx/openstack-ops/locale/chap/app_crypt.xml.ja.po:
```diff
@@ -0,0 +1,992 @@
+"tcpdump to start displaying the hex dump of the packet. We pinged between "
+"every combination of outside, controller, compute, and instance."
+msgstr ""
+"１つのクラウドコントローラーが全 compute ノードのゲートウェイの役割を果たして"
```
です、ます調になってます。ここの流れからだと、だ、である調にすべきです。
computeはコンピュートに統一すべきでしょうか?

takekawa commented on doc/src/docbkx/openstack-ops/locale/chap/app_crypt.xml.ja.po:
```diff
@@ -0,0 +1,992 @@
+"加えて、問題のインスタンスは毎晩非常に長いバックアップジョブを担っていた。"
+"「あの問題」（今では我々はこの障害をこう呼んでいる）はバックアップが行われて"
+"いる時には全く起こらず、我々がそれを無視できないところまでもうちょっと（数時"
+"間）だった。"
```
「我々がそれを無視できないところまでもうちょっと（数時間）だった。」

英文もみたのですが、この意味がよくわかりませんでした。。。

takekawa commented on doc/src/docbkx/openstack-ops/locale/chap/app_crypt.xml.ja.po:
```diff
@@ -0,0 +1,992 @@
+msgstr ""
+"パッチを当てた qemu と再現方法を携えて、我々は「あの問題」を最終的に解決した"
+"かを確認する作業に着手した。インスタンスにネットワーク負荷をかけてから丸48時"
+"間後、我々は確信していた。残りは記録である。あなたは、joe へのバグ報告を検索"
```
The rest is history.は慣用句のようです。
http://eow.alc.co.jp/search?q=The+rest+is+history
「その後のことは周知の通りです。」でしょうか。

takekawa commented on doc/src/docbkx/openstack-ops/locale/chap/app_crypt.xml.ja.po:
```diff
@@ -0,0 +1,992 @@
+"user affected."
+msgstr ""
+"私は Glance をチェックし、問題のイメージがユーザの作成したスナップショットで"
+"ある事に注目した。最終的に、それはグッドニュースだった。このユーザはユーザが"
```
このユーザはユーザが影響したイメージだけ持っていた。
は
このイメージの影響をうけるのはこのユーザのみだった。
ではないでしょうか?

amotoki committed:
表記の揺れを置換

「・」を削除
compute -> コンピュート
〜事 -> 〜こと
10GB/1GB NIC -> 10Gb/1Gb NIC (GB = Giga Byte, Gb = Giga bit)

amotoki committed:
レビューコメントを反映 & 何点か修正

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/app_crypt.xml.ja.po:
```diff
@@ -0,0 +1,992 @@
+"ウドコントローラーと全コンピュートノードで、各 OpenStack プロジェクトが異なる "
+"VLAN を持つことを意味する。パケットサイズ変更のため、ping の -s オプションを使"
+"用していた。パケットが全て戻ってくる時もあれば、パケットが出ていったきり全く戻って来ない"
+"時もあれば、パケットはランダムな場所で止まってしまう時もある、という状況だった。"
```
レビューコメントを反映したときに、翻訳を修正しましたが、
We watched as sometimes packets would fully return, sometimes they'd only make it out and never back in, and sometimes the packets would stop at a random point.
ですが、状況によって違う現象が見られたということだと思います。

「パケットが全て戻ってくる時もあれば、パケットが出ていったきり全く戻って来ない
時もあれば、パケットはランダムな場所で止まってしまう時もある、という状況だった。」としました。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/app_crypt.xml.ja.po:
```diff
@@ -0,0 +1,992 @@
+"タンスが固定 IP アドレスを更新するためにクラウドコントローラー（DHCP サー"
+"バー）に接続することを意味する。幾つかの理由で、このインスタンスはその IP アド"
+"レスを更新できなかった。インスタンスのログとクラウドコントローラー上のログを突き合わせ、"
+"並べてやりとりにしてみた。"
```
（更新した点についての補足です）
L.341 は "review" なので「受け取った」ではなく「見直した」としました。
L.345 は correlate は「突き合わせを行う」という意味だと思います。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/app_crypt.xml.ja.po:
```diff
@@ -0,0 +1,992 @@
+msgstr ""
+"この情報により、我々は問題が DHCP 実行に起因するものと確信した。何らかの理由で"
+"インスタンスが新しいIPアドレスを取得できず、その結果IPアドレスがなくなり、"
+"インスタンスは自分自身をネットワークから切り離した、と考えた。"
```
L.388-389に対応する翻訳を変更しました。確認願います。

amotoki commented on doc/src/docbkx/openstack-ops/locale/chap/app_crypt.xml.ja.po:
```diff
@@ -0,0 +1,992 @@
+"加えて、問題のインスタンスは毎晩非常に長いバックアップジョブを担っていた。"
+"「あの問題」（今では我々はこの障害をこう呼んでいる）はバックアップが行われて"
+"いる最中には起こらなかったが、（数時間たっていて）「あの問題」が起こるまで"
+"あと少しのところだった。"
```
（武川さんコメント再掲）
「我々がそれを無視できないところまでもうちょっと（数時間）だった。」
英文もみたのですが、この意味がよくわかりませんでした。。。

amotoki replied:
後ろまで読むと分かりますが、たくさんトラフィックを流し続けると時間が経過すると発生する問題です。
バックアップでトラフィックがたくさん流れていたが、そのときには「あの問題」は起こらなかったが、
あともう少しというところまでは来ていたということでしょう。
訳としては基本的に合っていますが、もう少し分かりやすくするとしたら、

意訳が入っていますが、こんな感じでしょうか。
ignore it の it は「あの問題」でしょう。問題を無視できない、ということは、
問題が起こってしまう、ということなので、そのようにしてみました。
exactly は when にかかっているはずなので、強調するために「行われている最中」としてみました。

「「あの問題」（今では我々はこの障害をこう呼んでいる）はバックアップが行われている最中には起こらなかったが、  （数時間たっていて）「あの問題」を無視できない状況になるまであと少しのところだった。」
